### ISSUE TITLE
Attribute: {type: email} is not a type for Sails v.011 per documentation

### ISSUE THREAD

**Issue by devongennuso** [balderdashy/sails-docs#551]
Sails Docs: http://sailsjs.org/documentation/concepts/models-and-orm/attributes

Type <b>email</b> is not listed as one of the attribute types. However, its listed as <b>{type: email}</b> in various places.

Attributes supported by Sails
string
text
integer
float
date
datetime
boolean
binary
array
json

**Comment by sailsbot** [comment]
Thanks for posting, @devongennuso.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
